tests/cdi/cdi_test.go

```diff
@@ -0,0 +1,135 @@
+package cdi_test
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// template parameters
+const (
+	pvcEPHTTPNOAUTHURL  = "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
+	pvcName        = "golden-pvc"
+	vmName         = "test-vm"
+	rawPVCFilePath = "tests/cdi/golden-pvc.yml"
+	rawVMFilePath  = "tests/cdi/test-vm.yml"
+	sepP = "-p"
+)
+
+var KubeVirtOcPath = ""
+
+var _ = Describe("CDI populate pvc create binding, running vm", func() {
+	flag.Parse()
+	SkipIfNoOc()
+	dstPVCFilePath := "/tmp/test-pvc"
+	dstVMFilePath := "/tmp/test-vm"
+
+	overrideTemplateFromParameters := func(srcFilePath, dstFilePath string, params ...string) string {
```

Comment:
This is just processing a template, not overriding it, right?  Perhaps we should change the function name?

Reply:
yes, I change it to `processTemplateWithParameters`, more accurately